docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -14,305 +14,194 @@ weight: 700
+The result of the JMESPath expression should be a string array of groups.
 
-Furthermore, Grafana will check for the presence of at least one of the teams specified via the `team_ids` configuration option using the [JMESPath](http://jmespath.org/examples.html) specified via the `team_ids_attribute_path` configuration option. The JSON used for the path lookup is the HTTP response obtained from querying the Teams endpoint specified via the `teams_url` configuration option (using `/teams` as a fallback endpoint). The result should be a string array of Grafana Team IDs. Using this setting ensures that only certain teams is allowed to authenticate to Grafana using your OAuth provider.
+You can limit access to Grafana to only members of a given group or list of groups by setting the `allowed_groups` option.
 
-You can limit access to only members of a given group or list of groups by setting the `allowed_groups` option.
+### Teams
 
-### Login
+OAuth2 provider teams are extracted using [JMESPath](http://jmespath.org/examples.html) expression from the `team_ids_attribute_path` configuration option.
+The expression is evaluated against the JSON data obtained from Teams endpoint, which is specified via the `teams_url` configuration option.
+The result should be a string array of OAuth2 provider team IDs.
 
-Customize user login using `login_attribute_path` configuration option. Order of operations is as follows:
+If `team_ids` configuration option is set, only users who are members of at least one of the specified teams will be able to authenticate to Grafana using your OAuth2 provider.
 
-1. Grafana evaluates the `login_attribute_path` JMESPath expression against the ID token.
-1. If Grafana finds no value, then Grafana evaluates expression against the JSON data obtained from UserInfo endpoint. The UserInfo endpoint URL is specified in the `api_url` configuration option.
+### SSL/TLS configuration
 
-You can customize the attribute name used to extract the ID token from the returned OAuth token with the `id_token_attribute_name` option.
+You can specify the SSL/TLS configuration used by the client.
 
-You can set the user's display name with JMESPath using the `name_attribute_path` configuration option. It operates the same way as the `login_attribute_path` option.
+- Set `tls_client_cert` to the path of the certificate.
+- Set `tls_client_key` to the path containing the key.
+- Set `tls_client_ca` to the path containing a trusted certificate authority list.
 
-> **Note:** `name_attribute_path` is available in Grafana 7.4+.
+`tls_skip_verify_insecure` controls whether a client verifies the server's certificate chain and host name. If it is true, then SSL/TLS accepts any certificate presented by the server and any host name in that certificate. _You should only use this for testing_, because this mode leaves SSL/TLS susceptible to man-in-the-middle attacks.
 
 ### PKCE
```

Comment:
I'm not sure why this is included in this topic. is this important for the user to know?